modules/deploy/pages/deployment-option/cloud/managed-connectors/transforms.adoc

```diff
@@ -0,0 +1,719 @@
+To use it make sure to remove the `_schema` topic from the topic exclude list.
+
+=== Example
+
+Sample configuration:
+
+----
+"transforms": "schema-replicator",
+"transforms.schema-replicator.type": "com.redpanda.connectors.transforms.SchemaRegistryReplicator"
+----
+
+== SetSchemaMetadata
+
+SetSchemaMetadata SMT sets the schema name, version or both on the record's key (`org.apache.kafka.connect.transforms.SetSchemaMetadata$Key`) or value (`org.apache.kafka.connect.transforms.SetSchemaMetadata$Value`)schema.
+
+=== Configuration
+
+|=== 
+| Property key | Description
+
+| `schema.name`
+| Schema name to set.
+
+| `schema.version`
+| Schema version to set.
+|===
+
+=== Example
+
+Sample configuration:
+
+----
+"transforms": "SetSchemaMetadata",
+"transforms.SetSchemaMetadata.type": "org.apache.kafka.connect.transforms.SetSchemaMetadata$Value",
+"transforms.SetSchemaMetadata.schema.name": "transaction-value"
+"transforms.SetSchemaMetadata.schema.version": "3"
+----
+
+== TimestampConverter
+
+The TimestampConverter SMT converts timestamps between different formats such as Unix epoch, strings, and Connect Date/Timestamp types. Applies to individual fields or to the entire value.
+
+Use the concrete transformation type designed for the record key (`org.apache.kafka.connect.transforms.TimestampConverter$Key`) or value (`org.apache.kafka.connect.transforms.TimestampConverter$Value`).
+
+=== Configuration
+
+|=== 
+| Property key | Description
+
+| `field`
+| The field containing the timestamp, or empty if the entire value is a timestamp. Default: `""`.
+
+| `target.type`
+| The desired timestamp representation: `string`, `unix`, `Date`, `Time`, or `Timestamp`.
+
+| `format`
+| A `SimpleDateFormat`-compatible format for the timestamp. Used to generate the output when `target.type=string` or used to parse the input if the input is a string. Default: `""`.
+
+| `unix.precision`
+| The desired Unix precision for the timestamp: seconds, milliseconds, microseconds, or nanoseconds. Used to generate the output when type=unix or used to parse the input if the input is a Long. Note: This SMT will cause precision loss during conversions from, and to, values with sub-millisecond components. Default: `milliseconds`.
+
+|===
+
+=== Example
+
+Sample configuration:
+
+----
+"transforms": "TimestampConverter",
+"transforms.TimestampConverter.type": "org.apache.kafka.connect.transforms.TimestampConverter$Value",
+"transforms.TimestampConverter.field": "last_login_date",
+"transforms.TimestampConverter.format": "yyyy-MM-dd",
+"transforms.TimestampConverter.target.type": "string"
+----
+
+Before: `1702041416`
+
+After: `2023-12-08`
+
+== TimestampRouter
+
+The TimestampRouter SMT updates the record's topic field as a function of the original topic value and the record timestamp. This is mainly useful for sink connectors, since the topic field is often used to determine the equivalent entity name in the destination system (e.g. database table or search index name).
```

Comment:
```suggestion
TimestampRouter SMT updates the record's topic field as a function of the original topic value and the record timestamp. This is mainly useful for sink connectors, because the topic field is often used to determine the equivalent entity name in the destination system (for example, a database table or search index name).
```